src/lando/pushlog/models/commit.py

```diff
@@ -239,13 +239,52 @@ class Tag(models.Model):
         on_delete=models.SET_NULL,
     )
 
+    _scm_commit: CommitData | None = None
+
+    @staticmethod
+    def for_scm_commit(repo: Repo, name: str, scm_commit: CommitData):  # noqa: ANN205
+        """Create a partial Tag object, based on CommitData not present in the DB
```

Comment:
Nit: first line of a docstring should be a single line, followed by a newline.

Reply:
I need to be use words with fewer letters 😅